EIPS/eip-bit_based_permission.md

```diff
@@ -12,17 +12,27 @@ created: 2023-02-27
 
 ## Abstract
 
-This EIP offers a standard to build permission and role system based on bits. Each permission is represented by a single bit in `uint256` from which we can defined up to `256` permissions and `2²⁵⁶` roles. This approach use bitwise operator and bitmask to determine the access right which is much more efficient and flexible than `string` comparison or `keccak256(string)`. We are able to specify the importance of permission based on the bit order.
+This EIP offers a standard for building a bit-based permission and role system. Each permission is represented by a single bit in `uint256` from which we can defined up to `256` permissions and `2²⁵⁶` roles. We are able to specify the importance of permission based on the order of the bits.
 
 ## Motivation
 
-**Role is a combination of multiple permissions**, we can use `OR` operator to combine new role based on multiple permissions, since permissions were defined as a power of two.
+Currently permission and access control are done using `string` or `keccak256(string)` comparaisons (see [ERC-5982](./eip-5982.md)). This is a very user friendly way to do this.
+But by using bitwise and bitmask operations to determine access rights, we get a lot of benefits such as efficiency, flexibility and others, let's walk through them.
```

Comment:
> let's walk through them.

I think, Ethereum ppl won't prefer this style. They suppose everything in present sentence and have a clear mean without influence and style.